### DISCUSSION TITLE
Does Internationalization includes also Right-To-Left support?

### DISCUSSION THREAD

**Discussion by liorbalmas** [ant-design-blazor/ant-design-blazor#1096] [General]
Thanks

**Comment by ElderJames** [comment]
Thank you for contacting us @liorbalmas

We haven't support the RTL yet right now, but we would do that like Ant Design.

Are you interesting to help us to support it?

**Reply by liorbalmas** [reply]
Hi @ElderJames , 
Actually, I'm pretty new to Ant. (I came across it while looking for a UI framework for some project).
But surly I'd love to help in whatever I can.
Please feel free to contact me on this matter.
Best Regards

**Reply by ElderJames** [reply]
Great! You just need to study how Ant Design achieves RTL and feel free to submit PRs.

**Comment by ElderJames** [comment]
I just created a Pull Request for RTL #1238

**Comment by ElderJames** [comment]
Now, the RTL support was added in the feature branch. Please visit the preview site and help us to improve more.

https://preview-1238-antblazor.surge.sh/

**Comment by ElderJames** [comment]
Hi @liorbalmas 0.8.0 has supported RTL.

**Reply by mo3in** [reply]
thanks for supporting RTL but right now in some components we don't have full RTL support yet. likes `paddings`, `margins` is not correct in `RTL`.

- `Menu` component have wrong `margin` for icons. 
- `Pagination` , `Cascader ` icons have wrong direction.
- dialogs like `Drawer`, `Modals` when you open them, the page content have some movement (because of page scrollbar).
- `carousel` RTL mode you cant see anything
- input `Pre / Post tab` , `Search Input` border left and right is not correct.

**Reply by ElderJames** [reply]
Thanks for the research @mo3in .
I am inexperienced in this area, would you be interested in helping to fix it?
